src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -2612,8 +2612,7 @@ static QVariant fcnGeomMakeValid( const QVariantList &values, const QgsExpressio
   }
   catch ( QgsNotSupportedException & )
   {
-    parent->setEvalErrorString( QObject::tr( "The make_valid parameters require a newer GEOS library version" ) );
-    return QVariant();
+    parent->setEvalErrorString( QObject::tr( "The make_valid parameters require a newer GEOS library version" ) ); return QVariant();
```

Comment:
why this change ? The original 2 line one is more readable

Reply:
If it's me, it was unintentional.